hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -26,6 +26,8 @@ public record ContractsConfig(
                 boolean noncesExternalizationEnabled,
         @ConfigProperty(defaultValue = "false") @NetworkProperty boolean enforceCreationThrottle,
         @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerSec,
+        @ConfigProperty(defaultValue = "15000000") @NetworkProperty long maxGasPerSecBackend,
```

Comment:
Do these need to be independently configurable if they're using the same value in mainnet?

Reply:
That is the idea.  We want to have separate values so that they can be independently configured on mainnet eventually.  We will likely want to have the front-end throttle limit be higher than the backend-throttle limit so that the it is the backend throttle that is actually binding.